docs/articles_en/openvino-workflow-generative/inference-with-genai/inference-with-genai-on-npu.rst

```diff
@@ -244,11 +244,13 @@ reusing them for future pipeline runs.
 CACHE_DIR
 -----------------------------------------------------------------------------------------------
 
-``CACHE_DIR`` operates similarly to the older ``NPUW_CACHE_DIR``, except for two differences:
+``CACHE_DIR`` operates similarly to the older ``NPUW_CACHE_DIR``, except for differences below:
 
 * It creates a single ".blob" file and loads it faster.
-* It stores all model weights inside the blob, making it much bigger than individual compiled
-  schedules for model's subgraphs stored by ``NPUW_CACHE_DIR``.
+* By default the produced blob is weightless, so either original weights file or ``ov::Model``
```

Comment:
I think it makes sense here to state early that the produced blob is defined by the `CACHE_MODE`. So it is not by ours default the blob is weightless, but the default `CACHE_MODE` is `OPTIMIZE_SIZE` which makes it weightless.

Reply:
Good point, fixed